src/Compilers/CSharp/Test/Semantic/Semantics/ObjectAndCollectionInitializerTests.cs

```diff
@@ -2826,14 +2826,14 @@ static void Main(string[] args)
         var d = /*<bind>*/new Dictionary<object, object>()
         {
             {""s"", 1 },
-        var x = 1/*</bind>*/;
-    }
+        var x = 1;
+    }/*</bind>*/
```

Comment:
i recommend not changing existing test inputs.  It means we can't tell if the PR affected the output of some scenario.  Instead, keep the old test and add a new test with a new input.  THanks!

Reply:
Sadly I was unable to keep the original input in this case. Since the expression is now read beyond the `;` till the next `}` the test suite would start throwing exceptions. My understanding is that it couldnt find the expression ending with `</bind>` as the expression now ends at the next `}` I moved the `</bind>` there.